GoogleTestAdapter/Tests.Common/ResultChecker/TrxResultChecker.cs

```diff
@@ -61,7 +61,8 @@ public void RunTestsAndCheckOutput(string typeName, string arguments, string tes
                 Assert.Inconclusive($"First time this test runs, created golden file - check for correctness! File: {goldenFile}");
             }
             CheckIfFileIsParsable(goldenFile);
-            if (!CompareXmlFiles(goldenFile, transformedResultFile, out string diffFile))
+            string diffFile;
```

Comment:
OK, for now I will leave the following be:
 |   | remove C# 7.0 features |   |   | 048822d
 |   | Fix SampleTests (/DEBUG vs /DEBUG:FULL) |   |   | b9e1303
 |   | Fix end-to-end tests for VS 2015. … |   |   | 1eefc9a
and reverted the rest.